include/gcc/riscv64/ck_f_pr.h

```diff
@@ -0,0 +1,134 @@
+/* DO NOT EDIT. This is auto-generated from feature.sh */
```

Comment:
FYI this is a lie. I could not get `feature.sh` to output anything useful so I constructed and verified the contents of this header manually.

Basically, `feature.sh` requires building one of the regression test programs, but compilation failed without something being present in this header already. Even after working around this the script only output ~5 `#defines`.